### DISCUSSION TITLE
Display profile name in SFTP Explorer

### DISCUSSION THREAD

**Discussion by EizEddin** [Natizyskunk/vscode-sftp#141] [Ideas]
If we are using multiple profiles like in the screenshot, is there a way to display the active remote server/profile name in the SFTP Explorer so we know which remote server we are browsing?

![image](https://user-images.githubusercontent.com/3897959/161382547-b3d23d77-b1de-45f9-bfd4-d868f074433b.png)

**Comment by Natizyskunk** [comment]
@EizEddin, This seems defintely a great option to add.
I'll take a look at it when I've more time 😊

**Reply by Natizyskunk** [reply]
@satiromarra Maybe some help ?

**Reply by satiromarra** [reply]
It is an improvement pending to move.

**Reply by Natizyskunk** [reply]
@satiromarra hey, is there some news about this feature ? 🙂

**Reply by Natizyskunk** [reply]
@EizEddin Could you open an issue respecting [this](https://github.com/Natizyskunk/vscode-sftp/issues/new?assignees=&labels=&projects=&template=feature_request.md&title=) template please ?
